gui/network/migrations/0006_globalconfiguration_gc_domains.py

```diff
@@ -0,0 +1,25 @@
+# -*- coding: utf-8 -*-
+# Generated by Django 1.10.7 on 2017-08-23 11:07
+from __future__ import unicode_literals
+
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
+    dependencies = [
+        ('network', '0005_vlan_vlan_pcp')
+    ]
+
+    operations = [
+        migrations.AddField(
+            model_name='GlobalConfiguration',
+            name='gc_domains',
+            field=models.TextField(
+                max_length=256,
+                verbose_name="Additional domains",
+                default='',
+                blank=True,
+                help_text="Delimit domains by spaces."
```

Comment:
"Additional domains to be searched can be entered here, separated by spaces. Be aware that adding search domains can cause slow DNS lookups."